components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -85,6 +85,14 @@
  */
 @property(nonatomic, nonnull, readonly, copy) UIFont *overline;
 
+@optional
+
+/**
+ Whether this typography scheme adjusts the returned fonts based on the device's Dynamic Type
```

Comment:
Looks like this description is incorrect. All of the font values appear to use the `default` category size. Perhaps this is mean to read more like, "Whether consumers of this typography scheme should used scaled versions of fonts or not."?

Reply:
Comment updated.